engine.json

```
@@ -129,7 +129,8 @@
         "Templates/PythonToolGem",
         "Templates/RemoteRepo",
         "Templates/ScriptCanvasNode",
-        "Templates/UnifiedMultiplayerGem"
+        "Templates/UnifiedMultiplayerGem",
+        "Templates/ScriptOnlyProject"
```

The templates list are kept in alphabetical order.
So "ScriptOnlyProject" should be above "UnifiedMultiplayerGem"
```suggestion
        "Templates/ScriptOnlyProject",
        "Templates/UnifiedMultiplayerGem"
```